test/e2e/user_input_test.go

```diff
@@ -19,26 +19,27 @@ import (
 )
 
 func waitCluster(input model.UserInputs, generation string) {
-	Eventually(kube.GetGeneration(input.Namespace, input.Clusters[0].GetClusterNameResource())).Should(Equal(generation))
-	Eventually(
-		kube.GetStatusCondition(input.Namespace, input.Clusters[0].GetClusterNameResource()),
+	EventuallyWithOffset(
+		1, kube.GetStatusCondition(input.Namespace, input.Clusters[0].GetClusterNameResource()),
 		"45m", "1m",
 	).Should(Equal("True"), "Kubernetes resource: Cluster status `Ready` should be True")
 
-	Eventually(kube.GetK8sClusterStateName(
+	ExpectWithOffset(
+		1, kube.GetGeneration(input.Namespace, input.Clusters[0].GetClusterNameResource()),
+	).Should(Equal(generation))
+	ExpectWithOffset(1, kube.GetK8sClusterStateName(
 		input.Namespace, input.Clusters[0].GetClusterNameResource()),
-		"45m", "1m",
 	).Should(Equal("IDLE"), "Kubernetes resource: Cluster status should be IDLE")
 
-	Expect(
-		mongocli.GetClusterStateName(input.ProjectID, input.Clusters[0].Spec.Name),
+	ExpectWithOffset(
```

Comment:
[q] what does `ExpectWithOffset` do?

Reply:
if we have test asserts in another package, and the test fails on this assert, it shows another package code line, and we usually want to know the test code line

Reply:
oh I see - very useful feature, will use in integration tests as well, thanks!